examples/vision/cct.py

```diff
@@ -384,4 +384,9 @@ def run_experiment(model):
 
 The authors also demonstrate the performance of Compact Convolutional Transformers on
 NLP tasks and they report competitive results there.
+
+Example available on HuggingFace:
```

Comment:
See previous comment. I think you need a return between the intro text and the table to make the link table render correctly. That's what I'm seeing in the notebook. Can you add that and regenerate?